core/src/apps/ethereum/layout.py

```
@@ -372,10 +372,10 @@ async def confirm_typed_value_onekey(
     type_name = get_type_name_onekey(field)
 
     if array_index is not None:
-        title = limit_str(".".join(parent_objects + [name]))
+        title = ".".join(parent_objects + [name])
```

Ruff (0.8.2)

Consider `[*parent_objects, name]` instead of concatenation

Replace with `[*parent_objects, name]`

(RUF005)